src/benchmark.jl

```diff
@@ -0,0 +1,40 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
```

Comment:
or compare.jl ? 

I've got a new theory: we should use a naming as simple as possible!